Tools/Scripts/webkitpy/layout_tests/run_webkit_tests_integrationtest.py

```diff
@@ -619,6 +619,78 @@ def test_crash_with_stderr(self):
             sorted(list(actual_dictionary['tests']['failures']['unexpected'])),
         )
 
+    def test_wpt_tests(self):
+        host = MockHost()
+        _, regular_output, _ = logging_run(['imported/w3c/web-platform-tests/'], tests_included=True, host=host)
+        actual_dictionary = json.loads(host.filesystem.read_text_file('/tmp/layout-test-results/full_results.json')[len('ADD_RESULTS('):-2])
+        expected_dictionary = {
+            'tests': {
+                'imported': {
+                    'w3c': {
+                        'web-platform-tests': {
+                            'some': {
```

Comment:
Let's also add a 'crashtests' folder for testing.

Reply:
Sure.